homeassistant/components/tado/binary_sensor.py

```diff
@@ -260,3 +270,4 @@ def _async_update_zone_data(self):
                 self._tado_zone_data.open_window
                 or self._tado_zone_data.open_window_detected
             )
+            self._state_attributes = self._tado_zone_data.open_window_attr
```

Comment:
Please set the state attribute keys explicitly to allow us to see what they are and their formatting.